src/DocumentationBrowserViewExtension/MarkdownHandler.cs

```diff
@@ -14,24 +14,10 @@ internal class MarkdownHandler : IDisposable
         private const string SYNTAX_HIGHLIGHTING = "Dynamo.DocumentationBrowser.Docs.syntaxHighlight.html";
         private readonly Md2Html converter = new Md2Html();
 
-
-        private static MarkdownHandler instance;
-        /// <summary>
-        /// Gets MarkdownHandler instance
-        /// </summary>
-        internal static MarkdownHandler Instance
-        {
-            get
-            {
-                if (instance is null) { instance = new MarkdownHandler(); }
-                return instance;
-            }
-        }
-
         /// <summary>
         /// Constructor
         /// </summary>
-        private MarkdownHandler()
+        internal MarkdownHandler()
```

Comment:
why is this needed?

Reply:
ahh, to keep a public constructor from being created automatically?

Reply:
Correct.